### PR TITLE
Clear Data Model when saveCard is fired.

### PR DESCRIPTION
**Pull request by hborrelli1**
- Added the outfit title to newOutfit object on saveOutfit. 
- Added function to re-create new newOutfit object after saving outfit.
- Added CSS to keep bear at the top of the page when outfits list gets long.
- Created a resetDataModel function to create a new object after the current object is saved and added to localStorage

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -134,9 +134,12 @@ function disableSaveButton(event){
 function saveOutfit() {
   var outfitName = saveOutfitInput.value;
   var savedOutfitCard = `<button class="button-style">${outfitName}<img class="close-btn" src="assets/close.svg" alt="Close"></button>`;
+  newOutfit.title = outfitName;
   savedOutfitsList.insertAdjacentHTML('beforeend', savedOutfitCard);
   window.localStorage.setItem(outfitName, savedOutfitCard);
+  outfits.push(newOutfit);
   clearInputField(saveOutfit);
+  resetDataModel();
```
Comment (hborrelli1) [inline main.js:142]: Adding the title to the object before saving card. Then firing the resetDataModel function.

File: main.js [hunk 2]
```diff
@@ -169,3 +171,7 @@ function buttonReset(event) {
     }
   }
 }
+
+function resetDataModel() {
+  newOutfit = new Outfit(Date.now());
+}
```
Comment (hborrelli1) [inline main.js:177]: This creates a new object with a new date that gets fired after we save the current outfit to local storage

File: styles.css [hunk 3]
```diff
@@ -33,6 +33,7 @@ h3 {
 main {
   display: flex;
   padding: 10px 20px 20px 20px;
+  align-items:flex-start;
```
Comment (hborrelli1) [inline styles.css:36]: This keeps the columns content at the top of the containers when another section becomes longer.

File: main.js (hunk 1 above)
Comment (jeremiahblackol) [reply main.js:142]: awesome. looks really clean

**Pull request merged**